draft-packages/form/KaizenDraft/Form/SearchField/SearchField.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react"
+
+import { InputSearch } from "../Primitives/InputSearch"
+import { FieldGroup, Label } from ".."
+export interface SearchFieldProps
+  extends Omit<React.InputHTMLAttributes<HTMLInputElement>, "className"> {
+  inputRef?: React.RefObject<HTMLInputElement>
+  reversed?: boolean
+  loading?: boolean
+  primary?: boolean
+  secondary?: boolean
+  variant?: "default" | "prominent"
+  labelText?: React.ReactNode
+}
+
+const SearchField: React.FunctionComponent<SearchFieldProps> = ({
+  id,
+  disabled = false,
+  placeholder,
+  inputRef,
+  reversed = false,
+  loading,
+  primary,
+  secondary,
+  variant,
+  value,
+  defaultValue,
+  labelText,
+  ...genericInputProps
+}) => (
+  <FieldGroup
+    id={`${id}-field-group`}
+    inline={false}
+    automationId={`${id}-field-group`}
+  >
+    <Label
+      id={`${id}-field-label`}
+      automationId={`${id}-field-label`}
+      htmlFor={`${id}-field-textarea`}
```

Comment:
This needs to match the `id` sent to `InputSearch`. It'd be worth chucking it into a variable to make sure they stay the same.